content/blog/2016/2016-02-24-jenkins-security-update.md

```diff
@@ -0,0 +1,12 @@
+---
+:layout: post
+:title: Jenkins security updates
+:tags:
+- general
+- core
+- jenkinsci
```

Comment:
Not sure whether I can use tags that didn't exist on the old site, so I went with what a previous announcement used.